app/service/data_svc.py

```diff
@@ -201,3 +198,19 @@ def __init__(self, dao, utility_svc):
 
     async def update(self, table, key, value, data):
         await self.dao.update(table, key, value, data)
+
+
+    """ SUPPORT """
+    async def _create_attack(self, technique, tactic):
+        entry = await self.dao.get('core_attack', dict(attack_id=technique['attack_id']))
+        if len(entry) == 0:
+            await self.dao.create('core_attack',
+                              dict(attack_id=technique['attack_id'], name=technique['name'], tactic=str("['"+tactic+"']")))
+            entry = await self.dao.get('core_attack', dict(attack_id=technique['attack_id']))
+        else:
+            s_tactics = eval(entry[0]['tactic'])
+            if tactic not in s_tactics:
+                s_tactics.append(tactic)
+                await self.dao.update(table='core_attack', key='attack_id', value=technique['attack_id'],
+                                      data=dict(tactic=str(s_tactics)))
```

Comment:
I think that s_tactics should be stored differently here casting the list to a string will make it difficult to work with across other languages and it's multiple values stored as a string so it would always require string parsing to work with. also making querying difficult.

Reply:
I've turned it into a json representation of a list. Should be a bit easier to interact with.